packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_misc.spec.ts

```diff
@@ -7,8 +7,97 @@ context('multi-domain misc', { experimentalSessionSupport: true, experimentalMul
 
   it('verifies number of cy commands', () => {
     // @ts-ignore
-    expect(Object.keys(cy.commandFns).length).to.equal(86,
-      'The number of cy commands has changed. Please ensure any newly added commands are also tested in multi-domain.')
+    expect(Object.keys(cy.commandFns)).to.eql(
```

Comment:
I got an error here and didn't know which commands were new.

Reply:
I don't think this will be a problem once multi-domain merges to develop as the person adding the new command would see the failure before anyone else.

Reply:
Probably a personal preference, but in this case I think the file is more readable if there are more items on a single line:
```
[
  'check', 'uncheck', 'click', 'dblclick', 'rightclick', 'focus', 'blur', 'hover', 'scrollIntoView', 'scrollTo', 'select',  
  'selectFile', 'submit', 'type', 'clear', 'trigger', 'as', 'ng', 'should', 'and', 'clock', 'tick', 'spread', 'each', 'then',
  'invoke', 'its', 'getCookie', 'getCookies', 'setCookie', 'clearCookie', 'clearCookies', 'pause', 'debug', 'exec', 'readFile',
  'writeFile', 'fixture', 'clearLocalStorage', 'url', 'hash', 'location', 'end', 'noop', 'log', 'wrap', 'reload', 'go', 'visit',
  'focused', 'get', 'root', 'contains', 'within', 'shadow', 'request', 'session', 'screenshot', 'task', 'find', 'filter', 'not',
  'children', 'eq', 'closest', 'first', 'last', 'next', 'nextAll', 'nextUntil', 'parent', 'parents', 'parentsUntil', 'prev',
  'prevAll', 'prevUntil', 'siblings', 'wait', 'title', 'window', 'document', 'viewport', 'server', 'route', 'intercept', 'switchToDomain',
],
```